src/v0/destinations/sfmc/transform.js

```diff
@@ -234,10 +258,12 @@ const responseBuilderSimple = async (message, category, destination) => {
     if (typeof message.event !== 'string') {
       throw new ConfigurationError('Event name must be a string');
     }
+    if (hashMapEventDefinition[message.event.toLowerCase()]) {
+      return responseBuilderForMessageEvent(message, subDomain, authToken, hashMapEventDefinition);
```

Comment:
Should we add a component test to cover our changes ?

Reply:
Addressed.